spec/factories/status_factory.rb

```diff
@@ -16,4 +16,8 @@
     state { :withdrawn }
     association :details, factory: :withdrawal
   end
+
+  trait :published do
+    state { :published }
+  end
```

Comment:
We should move this into the factory block (and kill off the rogue withdrawn one)